src/ontology/edits/oeo-shared.omn

```diff
@@ -128,8 +183,30 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/450",
         <http://purl.obolibrary.org/obo/BFO_0000015>
     
     
-Class: <http://purl.obolibrary.org/obo/BFO_0000015>
+Class: OEO_00030031
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "start time is a zero-dimensional temporal region that indicates the beginning of a one-dimensional temporal region.",
```

Comment:
Please start the sentence with a capital letter.

Reply:
I would also add an "A" in the beginning, so "A start time..." (and analogous for the others).